components/brave_new_tab_ui/containers/app.tsx

```diff
@@ -46,11 +48,13 @@ function DefaultPage (props: Props) {
     <NewTabPage
       newTabData={newTabData}
       todayData={braveNewsData}
+      braveVPNData={braveVPNData}
       gridSitesData={gridSitesData}
       actions={actions}
       saveShowBackgroundImage={PreferencesAPI.saveShowBackgroundImage}
       saveShowRewards={PreferencesAPI.saveShowRewards}
       saveShowBraveTalk={PreferencesAPI.saveShowBraveTalk}
+      saveShowBraveVPN={PreferencesAPI.saveShowBraveVPN}
```

Comment:
This shouldn't be passed as a prop - you can just access it directly when you need it, so there's no need to drill these props all the way down.

Reply:
If you use the `usePref` hook instead you won't even need to do that!